dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractJdbcConnectorTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Google LLC
```

Comment:
Note: we should update the template to `2023`. Not required in this PR.

Reply:
Noted, Thanks!